spec/functions/extlib/dump_args_spec.rb

```diff
@@ -1,30 +1,32 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 require 'json'
 
 describe 'extlib::dump_args' do
   it 'exists' do
-    is_expected.not_to be_nil
+    expect(subject).not_to be_nil
   end
 
   it 'requires an argument' do
-    is_expected.to run.with_params.and_raise_error(ArgumentError)
+    expect(subject).to run.with_params.and_raise_error(ArgumentError)
   end
 
   context 'when called with a hash' do
     let(:hash) { { 'param_one' => 'value', 'param_two' => 42 } }
 
     it 'puts pretty JSON' do
-      allow(STDOUT).to receive(:puts).with(JSON.pretty_generate(hash))
-      is_expected.to run.with_params(hash)
+      allow($stdout).to receive(:puts).with(JSON.pretty_generate(hash))
+      expect(subject).to run.with_params(hash)
```

Comment:
I think I saw a cleaner way for this in https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexpectoutput but not sure how we should apply that here.